src/Features/Lsif/Generator/Microsoft.CodeAnalysis.LanguageServerIndexFormat.Generator.csproj

```diff
@@ -70,7 +70,7 @@
 
     <!-- Work around https://github.com/dotnet/msbuild/issues/7873; without this we won't have a new enough System.Memory and we'll get
```

Comment:
This MSBuild issue occured only on .NET Framework, and nothing changes when using Roslyn from it.